CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+<!-- Released section -->
+<!-- Don't change this section unless doing release -->
+
+## [1.35.0/0.60.0/0.29.0/0.15.0/0.10.0/0.8.0/0.7.0] - 2025-03-05
+
 This release is the last to support [Go 1.22].
 The next release will require at least [Go 1.23].
 
```

Comment:
`OTEL_SEMCONV_STABILITY_OPT_IN=http/dup` is listed in the `Added` section for `go.opentelemetry.io/contrib/instrumentation/github.com/gorilla/mux/otelmux`, but the `Changed` section for `go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin`. They should be put in the same section.

Reply:
good catch